client/src/components/layout/ReviewsList.js

```diff
@@ -1,17 +1,17 @@
 import React from "react"
 import ReviewTile from "./ReviewTile"
 
-const ReviewsList = ({reviews}) => {
+const ReviewsList = ({ reviews }) => {
     const listOfReviews = reviews.map((review) => {
         return (
-            <ReviewTile key={reviews.id} review={review}/>
+            <ReviewTile key={review.id} review={review} />
         )
     })
 
     return (
-        <ul>
-            {listOfReviews}
-        </ul>
+        <>
+            { listOfReviews }
```

Comment:
yes to space padding JS object, but not inside JSX
https://github.com/airbnb/javascript/tree/master/react#spacing